src/lib/storage/fixed_width_integer_vector.cpp

```diff
@@ -0,0 +1,35 @@
+#include "fixed_width_integer_vector.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+template <typename uintX_t>
+FixedWidthIntegerVector<uintX_t>::FixedWidthIntegerVector(size_t size) : _values(size) {}
```

Comment:
We weren't sure what approach to choose, but since we did not find a standard approach (compare: https://stackoverflow.com/questions/8928547/vector-initialization-or-reserve), were sure that we would need this capacity and would not execute push_back e.g we found this to be the more compact and direct solution